e2e_tests/cli_operations.py

```diff
@@ -36,3 +37,4 @@ def delete_bento(bento_name):
     ) as proc:
         delete_bento_stdout = proc.stdout.read().decode('utf-8')
     logger.info(delete_bento_stdout)
+    return delete_bento_stdout
```

Comment:
This function is called here https://github.com/bentoml/BentoML/blob/master/e2e_tests/yatai_server/test_postgres_s3.py#L47 and a value is assigned to the return, however this didn't return anything

Reply:
Thanks for surfacing this @kautukkundan 

@yubozhao could you look into it after this PR? It looks like this function was not used properly

Reply:
Yep